templates/addons/env/opensearch.yml

```diff
@@ -178,11 +188,11 @@ Resources:
         ZoneAwarenessConfig:
           AvailabilityZoneCount: 2
           #Fn::length always resolves to 1 despite there being subnets.
-          #Fn::Length: [ !Split [ ',', { 'Fn::ImportValue': !Sub '${App}-${Env}-PrivateSubnets' } ] ]
+          #Fn::Length: [ !Ref PrivateSubnet1, !Ref PrivateSubnet2 ]
```

Comment:
This is commented out ATM - is that intentional?

Reply:
It was commented out before. I supect it relates to some woes @lgarvey is looking at with this addon.

Reply:
We should probably have a ticket to come back and figure this one out.